docs/codeql/reusables/download-github-database.rst

```diff
@@ -0,0 +1,5 @@
+GitHub also stores CodeQL databases for over 200,000 repos on GitHub.com, which you can download using the API. The list of repos is constantly growing and evolving to make sure that it includes the most interesting codebases for security research.
+
+To download a database from GitHub.com using the `GitHub CLI <https://cli.github.com/manual/gh_api>`__, use the following command:: 
+
```

Comment:
Is there any way to find out whether or not a CodeQL database is available for a project, either in the GitHub UI or via a REST API call?